src/Datadog.Trace.ClrProfiler.Managed/Integrations/MongoDbIntegration.cs

```diff
@@ -131,6 +131,7 @@ public static class MongoDbIntegration
                        .Start(moduleVersionPtr, mdToken, opCode, methodName)
                        .WithConcreteType(wireProtocolType)
                        .WithParameters(connection, cancellationToken)
+                       .WithNamespaceAndNameFilters(ClrNames.Ignore, "MongoDB.Driver.Core.Connections.IConnection", ClrNames.CancellationToken)
```

Comment:
Just an observation, shouldn't the namespace and filters always match the `TargetSignatureTypes` on the `InterceptMethodAttribute`?

Reply:
Pretty much